src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts

```diff
@@ -267,6 +296,16 @@ export class PoolListComponent extends ListWithDetails implements OnInit {
     const emptyStat: PoolStat = { latest: 0, rate: 0, rates: [] };
 
     _.forEach(pools, (pool: Pool) => {
+      if (Array.isArray(pool.application_metadata)) {
+        const APP_LABELS: { [k: string]: string } = {
+          rbd: $localize`Block`,
+          rgw: $localize`Object`,
+          cephfs: $localize`Filesystem`
```

Comment:
we already have this mapping done somewhere else? so instead of duplication, shall we just reuse that? I think it was in the pool form

Reply:
Yes we are using it at multiple place but not sure what's out shared for something like this. Can  you help with it?

```
If we are hiding them by default always shall we just rename this as
Hide internal pools and then apply yes as the default?
```

Can we have just have `Internal pool` and the values can be `Hide` and `Show` instead of Yes and No, and `Hide`, could be default value. Thoughts?